writer.go

```diff
@@ -84,31 +84,45 @@ func (w *writeWorker) runWriteVLog(lc *y.Closer) {
 	for {
 		var r *request
 		select {
+		case task := <-w.ingestCh:
```

Comment:
Can put the task into a request so that we don't need another channel.
Then we don't need to `pollWriteCh` and flush them.